docs/core/tools/using-ci-with-cli.md

```diff
@@ -64,7 +64,17 @@ in the [CLI repo](https://github.com/dotnet/core/blob/master/Documentation/prere
 The below sections show examples of configurations using the mentioned CI SaaS offerings. 
 
 ### TravisCI
-**TODO**
+
+The [travis-ci](https://travis-ci.org/) can be configured to install the .NET Core SDK using the language `csharp` and `dotnet` key
```

Comment:
nit: missing period on the end of the sentence

also wondering if this would read better like this:
...using the `csharp` language and the `dotnet` key.